app/js/sessions.js

```diff
@@ -54,9 +54,13 @@ const enableAdblocker = () => {
         for (const name of Object.keys(defaultBlocklists)) {
             const list = path.join(__dirname, `../blocklists/${name}.txt`)
             try {
-                fs.copyFileSync(list, path.join(blocklistDir, `${name}.txt`))
+                // The copyFileSync function would preserve permissions
+                // (such as read-only)
```

Comment:
I don't think referring to the copy function actually adds a lot in this comment. Maybe a better comment would be `File is read and written separately to discard permission`. That is more clear to me about what is currently happening instead of saying why another way isn't suited. What are your thoughts on this?

Reply:
Ok, changed the comment.  Both explain why the next two lines shouldn't be refactored into a call to `copyFileSync`.